modules/nf-core/chromograph/main.nf

```diff
@@ -1,11 +1,12 @@
 process CHROMOGRAPH {
-    tag "$meta.id"
+    // $meta.id can be [] because autozyg is not required, so use all ids
+    tag "${[meta, meta2, meta3, meta4, meta5, meta6, meta7].collect { meta_map -> meta_map.id }.findAll { id -> id }.join('_') ?: 'chromograph'}"
```

Comment:
Do we really need to join all 7 of these? Can you really start the module with any of the 7?

Reply:
Weird, but ok.
How about you add a unique in there?
And what is `id -> id` doing?